draft-ietf-oauth-identity-chaining.md

```diff
@@ -58,7 +58,7 @@ informative:
 
 --- abstract
 
-This specification defines a mechanism to preserve identity and call chain information across trust domains that use the OAuth 2.0 Framework.
+This specification defines a mechanism to preserve identity information and federate authorization across trust domains that use the OAuth 2.0 Framework.
 
 --- middle
 
```

Comment:
I'm unfortunately not able to comment on the introduction section but we write there:
> Preserving this information is referred to as identity chaining. This document defines a mechanism for preserving identity chaining information across trust domains using a combination of ...

I believe with this change we should also change that section for example to
> This document referrers to this as "chaining" and defines a mechanism for preserving identity and authorization information across Domains.

Reply:
Yeah, agree we should also probably update that section somewhat.

Reply:
686f2d28f51470e86c8e7a0d90a989e6debd0b49 makes the update suggested by @arndt-s